lib/gds-sso/controller_methods.rb

```diff
@@ -1,11 +1,19 @@
 module GDS
   module SSO
+    class PermissionDeniedError < StandardError
+    end
+
     module ControllerMethods
-      class PermissionDeniedException < StandardError
+      # TODO: remove this for the next major release
```

Comment:
Could you maybe add a note to the commit message as to why this needs to be removed with the next major release?

Reply:
will do, thanks for the review :)